styles/styles.css

```diff
@@ -443,6 +442,11 @@ main img {
 
 /* START Section Styles START */
 
+main .section:not(.breadcrumb-container, .hero-horizontal-tabs-container, .carousel-container):first-of-type > div.section-container,
```

Comment:
The problem with large `not` selectors is that they become super specific and tough to override, thus forcing you to use `important!` . Is there are section variant already that applies this type of margin (mayb not exactly the same) that can be used instead?

Reply:
If we use a section style then the authors would need to add it specifically to the doc; we want the first section that isn't a hero, breadcrumbs, carousel etc to automatically have this spacing at the top

I checked (or at least tried to, sorry if some slipped) all unique pages, and some randomly selected ones for stuff like the newsroom etc that were all the same, and they all look good to me; I'll check with the others as well after we're happy with the implementation

I do agree that this is an ugly selector but I don't know if a section style would help

Reply:
Actually, an option just came to my mind, I could create/use a section style that would add the padding, and automatically add the class to the required sections (first section that follows the breadcrumb container that is not a hero, accordion etc)
wdyt?

Reply:
This seems to be very complex css selector

Reply:
>  I could create/use a section style that would add the padding, and automatically add the class to the required sections (first section that follows the breadcrumb container that is not a hero, accordion etc)

Yes if you see that this logic is applicable globally, you could programatically add the class .It'd have to be ensured that this happens before the page becomes visible to avoid CLS.